.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
do we also support 3.13? if so we can add it here for the tests?

Reply:
I would rather do it in a separate PR.